plasma_framework/migrations/1041_setup_vault.js

```diff
@@ -0,0 +1,187 @@
+/* eslint-disable no-console */
+/* eslint max-len: ["error", { "code": 500 }] */
+/* eslint object-curly-newline: ["error", "never"] */
+/* eslint-disable no-await-in-loop */
+const PlasmaFramework = artifacts.require('PlasmaFramework');
+const PaymentExitGame = artifacts.require('PaymentExitGame');
+const FeeExitGame = artifacts.require('FeeExitGame');
+const EthVault = artifacts.require('EthVault');
+const Erc20Vault = artifacts.require('Erc20Vault');
+const EthDepositVerifier = artifacts.require('EthDepositVerifier');
+const Erc20DepositVerifier = artifacts.require('Erc20DepositVerifier');
+const util = require('util');
+const childProcess = require('child_process');
+const fs = require('fs');
+const path = require('path');
+const config = require('../config.js');
+const pck = require('../package.json');
+
+const expectedBlockTime = 1000;
```

Comment:
expectedBlockTime is one second?

Reply:
it's more like how frequently we're willing to pull for results